src/main/java/stream/flarebot/flarebot/commands/moderation/PermissionsCommand.java

```diff
@@ -145,6 +143,19 @@ public void onCommand(User sender, GuildWrapper guild, TextChannel channel, Mess
                         group.getPermissions().clear();
                         MessageUtils.sendSuccessMessage("Cleared all permissions from the group: " + group.getName(), channel);
                         return;
+                    } else if (args[2].equalsIgnoreCase("move")) {
+                        if(args.length >= 4) {
+                            int pos = GeneralUtils.getInt(args[3], -1);
+                            if(pos < 1 || pos >= guild.getPermissions().getGroups().size()) {
+                                MessageUtils.sendWarningMessage("Invalid Page: " + args[3], channel);
+                                return;
+                            } else {
+                                pos = pos - 1;
```

Comment:
Is there even a point setting pos to -1 here? Just align the -1 and keep it as pos in L155

Reply:
ok

Reply:
I was going to do more with pos before but never happened